stackdriver/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+### 0.16.0 / 2019-08-20
```

Comment:
Wouldn't this be 0.15.5, not 0.16.0? To match the previous documentation-only tiny releases?